### PR TITLE
Fix Typos

### PR DESCRIPTION
**Pull request by standardAI**
This PR updates the configuration access, fixes typos, and trims trailing white spaces.

@stevhliu

### CODE REVIEW

**Comment by stevhliu** [comment]
The minor doc fixes LGTM, but I'd like to ping @sayakpaul for a second look at the config access changes

File: examples/community/stable_diffusion_ipex.py [hunk 1]
```diff
@@ -224,7 +224,7 @@ def get_input_example(self, prompt, height=None, width=None, guidance_scale=7.5,
         # 5. Prepare latent variables
         latents = self.prepare_latents(
             batch_size * num_images_per_prompt,
-            self.unet.in_channels,
+            self.unet.config.in_channels,
```
Comment (sayakpaul) [inline examples/community/stable_diffusion_ipex.py:227]: Can we tackle this in a separate PR?

File: src/diffusers/models/__init__.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
     _import_structure["autoencoders.autoencoder_tiny"] = ["AutoencoderTiny"]
     _import_structure["autoencoders.consistency_decoder_vae"] = ["ConsistencyDecoderVAE"]
     _import_structure["controlnet"] = ["ControlNetModel"]
-    _import_structure["dual_transformer_2d"] = ["DualTransformer2DModel"]
+    _import_structure["transformers.dual_transformer_2d"] = ["DualTransformer2DModel"]
```
Comment (sayakpaul) [inline src/diffusers/models/__init__.py:35]: Should be tackled in a separate PR.

**Review by sayakpaul — COMMENTED** [review]
Thanks for your contributions!

Can we keep this PR related to typos only and create other PRs for `config` access and init files?

**Comment by standardAI** [comment]
OK, even fixing a typo should be atomic; understood 👍.

**Comment by standardAI** [comment]
I planned this PR about configuration access. Additionally, I added fixing typos, because I counted them as nothing.

**Comment by sayakpaul** [comment]
Yeah it might appear to be easy but it really clogs the reviewing process. Hence my recommendation.

**Review by stevhliu — APPROVED** [review]
Thanks for the typo fixes! I'm going to merge this since there are a lot of changes already. For any other typos, please feel free to open a follow-up PR

**Comment by standardAI** [comment]
Thanks for giving feedback and merging!
